spec/stack_spec.rb

```diff
@@ -228,6 +228,40 @@
     stack.delete
   end
 
+  it "return list of stacks" do
+    class ResponseMockStack
+      attr_reader :name
+
+      def initialize(name)
+        @name = name
+      end
+
+      def stack_name
+        return @name
+      end
+    end
+
+    class ResponseMock
+      def stack_summaries
+        return [ResponseMockStack.new("first"), ResponseMockStack.new("second")]
+      end
+    end
+
+    client_double = double()
+    response = [ResponseMock.new(), ResponseMock.new()]
```

Comment:
Is there a reason why you left those `()` out?

Reply:
No, I just didn't look at them :D